main/forms.py

```diff
@@ -128,6 +131,29 @@ def clean_statement(self):
         return data
 
 
+class SpaceMembershipForm(ModelForm):
+    class Meta:
+        model = SpaceMembership
+        fields = ('fee', 'statement')
+        widgets = {
+            'fee': forms.NumberInput(attrs={'step': 0.25, 'min': 20.0})
+        }
+
+    # ensure fee is not less than £20.00
```

Comment:
Is this the minimum we agreed to?

Reply:
https://github.com/UKHackspaceFoundation/foundation/blob/master/structure.md - yes

Reply:
Thanks for reminding me.